components/cop_codegen/src/rpn_function.rs

```diff
@@ -62,3 +71,579 @@ impl RpnFunctionOpts {
         }
     }
 }
+
+const ARG_DEF: &str = "crate::coprocessor::dag::rpn_expr::function::ArgDef";
+const NULL: &str = "crate::coprocessor::dag::rpn_expr::function::Null";
+const RPN_FN_ARG: &str = "crate::coprocessor::dag::rpn_expr::function::RpnFnArg";
+const ARG: &str = "crate::coprocessor::dag::rpn_expr::function::Arg";
```

Comment:
Could these four turn into something like `ctx_type()` below?

Reply:
I don't think we really need these constants.